Model/AbstractJobs.php

```diff
@@ -156,8 +157,7 @@ public function setImmediateRetryAlreadyDone($entity, $entityId, $storeId, $flag
     public function isImmediateRetry($response, $entity, $entityId, $storeId)
     {
         $storeId = (int) $storeId;
-        /** @phpstan-ignore-next-line */
-        $responseCode = $response && $response->getData('status') ? $response->getData('status') : null;
+        $responseCode = is_object($response) && $response->getData('status') ? $response->getData('status') : null;
```

Comment:
It is just for making sure $response is an object and it has the "status" property. We can remove is_object check from here, but it is something like a double safety to avoid any run time errors in case $response is null by chance.